corehq/apps/registration/views.py

```diff
@@ -163,21 +161,26 @@ def register_new_user(self, data):
                     }
                 }
 
-            persona_fields = {}
-            if reg_form.cleaned_data['persona']:
-                persona_fields['buyer_persona'] = reg_form.cleaned_data['persona']
-                if reg_form.cleaned_data['persona_other']:
-                    persona_fields['buyer_persona_other'] = reg_form.cleaned_data['persona_other']
-                couch_user = CouchUser.get_by_username(reg_form.cleaned_data['email'])
-                if couch_user:
-                    update_hubspot_properties.delay(couch_user, persona_fields)
+            couch_user = CouchUser.get_by_username(reg_form.cleaned_data['email'])
```

Comment:
Why does this need to be earlier?

Reply:
It doesn't, it's coincidence that it moved up as I was rearranging the other code here.